README.md

```diff
@@ -110,31 +111,34 @@ npm install npm@latest -g
 
 ### Installation
 
-1. Sign up for your own non-relational document database at
-   [Firebase](https://firebase.google.com)
-2. Clone the repo
+1. Clone the repo
    ```sh
    git clone https://github.com/Stack-to-the-Future/atelier.git
    ```
-3. Install NPM packages
+1. Install NPM packages
    ```sh
    npm install
    ```
-4. Enter your port, database string, and API keys in `config.js` or `.env` file
+1. Enter your port, API URL, and Github Token in `.env` file
    ```sh
     PORT = 8080 (or port of your choosing)
-    DB_STRING = `your database URI`
+    URL = (your database URI)
+    TOKEN = (your github token)
    ```
-5. Run in development environment
+1. Run build command.
    ```sh
-   npm run dev
+   npm run build
+   ```
+1. Run in development environment
+   ```sh
+   npm run start
    ```
 
 ## Usage
 
-- How to
-- Use
-- Atelier
+Atelier is runs on the designated port and opens a webpage as soon as the npm run start command is utilized. It can also be accessed utilizing localhost:PORT directly in the browser.
```

Comment:
Hi Ming,
I am going to assume you meant  " Atelier run on..  " and not " Atelier is runs ... "